File: WebsiteAppRPG/Frontend/website-app-rpg-frontend/src/models/map.ts
export class Map {

    public mapId: number;

    public name: string;
    
    public width: number;
    
    public height: number;
    
    constructor(mapId: number, name: string, mapWidth: number, mapHeight: number) {
        this.mapId = mapId;
        this.name = name;
        this.width = mapWidth;
        this.height = mapHeight;
    }
}

File: WebsiteAppRPG/Core/Entities/Map.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace WebsiteAppRPG.Core.Entities
{
    [Table("Maps")]
    public class Map
    {
        [Key]
        [Column("MapID")]
        [JsonPropertyName("mapId")]
        public int MapId { get; set; }

        [Column("Name")]
        public string Name { get; set; }

        [Column("Width")]
        public int Width { get; set; }

        [Column("Height")]
        public int Height { get; set; }

        [ForeignKey("BiomeID")]
        [Column("BiomeID")]
        [JsonPropertyName("biomeId")]
        public int BiomeID { get; set; }

        public Map()
        {

        }

        public Map(string name, int width, int height, int biomeId)
        {
            Name = name;
            Width = width;
            Height = height;
            BiomeID = biomeId;
        }
    }
}
